### ISSUE TITLE
(Bug report) Hide Pane open and close in different window at the same time

### ISSUE THREAD

**Issue by fooyoo** [zadam/trilium#3296]
### Trilium Version

0.56-2

### What operating system are you using?

Windows

### What is your setup?

Local (no sync)

### Operating System Version

Windows 10 19044.2130

### Description

I noticed when I open a note in a new window, I click the "Hide Pane", see the pane is close while the pane in main window is closed two.  I expected the pane is only hide in the new opened window.
![hide-pane](https://user-images.githubusercontent.com/36770100/200552621-48c697fd-9b5e-4070-b0c9-6c4238ef8b48.gif)

**Comment by zadam** [comment]
Yeah, you're right. This is a rather design problem and not an easy bug to fix.

**Comment by fooyoo** [comment]
Understood, thanks a lot for the quick answer!

**Comment by eliandoran** [comment]
This is now supported.
